<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>E-Biblioteka</string>
  </property>
  <property name="windowIcon">
   <iconset resource="resources.qrc">
    <normaloff>:/icons/library.png</normaloff>:/icons/library.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout"/>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuPlik">
    <property name="title">
     <string>Plik</string>
    </property>
    <addaction name="actionNewList"/>
    <addaction name="actionOpenList"/>
    <addaction name="actionSaveList"/>
   </widget>
   <widget class="QMenu" name="menuProgram">
    <property name="title">
     <string>Program</string>
    </property>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuSortowanie">
    <property name="title">
     <string>Sortowanie</string>
    </property>
    <widget class="QMenu" name="menuAutor">
     <property name="title">
      <string>Autor</string>
     </property>
     <addaction name="actionSortAuthorAlpha"/>
     <addaction name="actionSortAuthorRev"/>
    </widget>
    <widget class="QMenu" name="menuTytu">
     <property name="title">
      <string>Tytuł</string>
     </property>
     <addaction name="actionSortTitleAlpha"/>
     <addaction name="actionSortTitleRev"/>
    </widget>
    <addaction name="menuAutor"/>
    <addaction name="menuTytu"/>
    <addaction name="actionSortFav"/>
    <addaction name="actionSortComp"/>
   </widget>
   <addaction name="menuPlik"/>
   <addaction name="menuSortowanie"/>
   <addaction name="menuProgram"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionOpenList"/>
   <addaction name="actionNewList"/>
   <addaction name="actionDeleteList"/>
   <addaction name="actionRenameList"/>
   <addaction name="actionSaveList"/>
   <addaction name="separator"/>
   <addaction name="actionAdd"/>
   <addaction name="actionDelete"/>
   <addaction name="actionMarkCompleted"/>
   <addaction name="actionFavourite"/>
   <addaction name="actionOpen"/>
  </widget>
  <action name="actionExit">
   <property name="text">
    <string>Wyjdź</string>
   </property>
  </action>
  <action name="actionOpenList">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/resources/icons/notebook.png</normaloff>:/icons/resources/icons/notebook.png</iconset>
   </property>
   <property name="text">
    <string>Otwórz listę</string>
   </property>
   <property name="toolTip">
    <string>Otwórz listę</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="text">
    <string>Zapisz</string>
   </property>
  </action>
  <action name="actionOpen">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/open.png</normaloff>:/icons/open.png</iconset>
   </property>
   <property name="text">
    <string>Otwórz</string>
   </property>
   <property name="toolTip">
    <string>Otwórz</string>
   </property>
  </action>
  <action name="actionAdd">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/plus.png</normaloff>:/icons/plus.png</iconset>
   </property>
   <property name="text">
    <string>Dodaj</string>
   </property>
   <property name="toolTip">
    <string>Dodaj dokument</string>
   </property>
  </action>
  <action name="actionDelete">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/minus.png</normaloff>:/icons/minus.png</iconset>
   </property>
   <property name="text">
    <string>Usuń</string>
   </property>
   <property name="toolTip">
    <string>Usuń dokument</string>
   </property>
  </action>
  <action name="actionRenameList">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/edit.png</normaloff>:/icons/edit.png</iconset>
   </property>
   <property name="text">
    <string>Zmień nazwę listy</string>
   </property>
   <property name="toolTip">
    <string>Zmień nazwę listy</string>
   </property>
  </action>
  <action name="actionMarkCompleted">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/mark.png</normaloff>:/icons/mark.png</iconset>
   </property>
   <property name="text">
    <string>Oznacz jako przeczytane</string>
   </property>
   <property name="toolTip">
    <string>Oznacz jako przeczytane</string>
   </property>
  </action>
  <action name="actionNewList">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/resources/icons/notebook--plus.png</normaloff>:/icons/resources/icons/notebook--plus.png</iconset>
   </property>
   <property name="text">
    <string>Nowa lista</string>
   </property>
  </action>
  <action name="actionDeleteList">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/resources/icons/notebook--minus.png</normaloff>:/icons/resources/icons/notebook--minus.png</iconset>
   </property>
   <property name="text">
    <string>Usuń listę</string>
   </property>
   <property name="toolTip">
    <string>Usuń listę</string>
   </property>
  </action>
  <action name="actionSaveList">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/resources/icons/disk-return-black.png</normaloff>:/icons/resources/icons/disk-return-black.png</iconset>
   </property>
   <property name="text">
    <string>Zapisz listę</string>
   </property>
   <property name="toolTip">
    <string>Zapisz listę</string>
   </property>
  </action>
  <action name="actionFavourite">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/heart.png</normaloff>:/icons/heart.png</iconset>
   </property>
   <property name="text">
    <string>Oznacz jako ulubione</string>
   </property>
   <property name="toolTip">
    <string>Oznacz jako ulubione</string>
   </property>
  </action>
  <action name="actionSortFav">
   <property name="text">
    <string>Ulubione</string>
   </property>
  </action>
  <action name="actionSortComp">
   <property name="text">
    <string>Ukończone</string>
   </property>
  </action>
  <action name="actionSortAuthorAlpha">
   <property name="text">
    <string>od A do Z</string>
   </property>
  </action>
  <action name="actionSortTitleAlpha">
   <property name="text">
    <string>od A do Z</string>
   </property>
  </action>
  <action name="actionSortAuthorRev">
   <property name="text">
    <string>od Z do A</string>
   </property>
  </action>
  <action name="actionSortTitleRev">
   <property name="text">
    <string>od Z do A</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
